Makefile

```diff
@@ -124,6 +124,7 @@ create-bindir:
 mig-agent: create-bindir
 	@echo building mig-agent for $(OS)/$(ARCH)
 	$(GO) build $(GOOPTS) -o $(BINDIR)/mig-agent-$(BUILDREV)$(BINSUFFIX) $(GOLDFLAGS) github.com/mozilla/mig/mig-agent
+	$(GO) build $(GOOPTS) -o $(BINDIR)/mig_agent_cfg tools/mig_agent_cfg.go
```

Comment:
This `mig_agent_cfg` binary uses underscores when all others seem to use hyphens. If it’s introduced newly in this patch, consider altering it to match if that’s appropriate.